crates/next-core/src/next_config.rs

```diff
@@ -192,6 +193,17 @@ impl NextConfigVc {
                 .unwrap_or_default(),
         ))
     }
+
+    #[turbo_tasks::function]
+    pub async fn image_config(self) -> Result<ImageConfigVc> {
+        let mut image_config = self.await?.images.clone();
+        // TODO is this the right place to do this?
+        image_config.remote_patterns.push(RemotePattern {
```

Comment:
I think we can drop this entirely now. It was just done because we couldn't load the next config, and we need to allow everything so that the things that were supposed to be allowed in the config would work. But now, we can load the config and know what was actually supposed to be allowed. If they didn't allow a host, then it should fail.

The reason it was still present in earlier PR revisions is because the web_source didn't load the config, but now that works, too.

Reply:
Sounds good. Follow-up PR